### ISSUE TITLE
Check if projects exist when opening montery

### ISSUE THREAD

**Issue by vegarringdal** [monterey-framework/monterey#76]
Just updated my app and started it, see that all folders of my test apps are still there.
Would be nice if we maybe tagged them with a red color&text telling user that they done exist.
No tile show show if you choose that project too.

that way you dont get errors like this:

![image](https://cloud.githubusercontent.com/assets/2901416/16888264/3f313228-4adf-11e6-97e7-acc87c22946e.png)

**Comment by adriatic** [comment]
Interesting timing - I just finished swapping my mac's hard disk with SSD and got the fastest possible macbook pro on my 5 years old laptop.

So, I cloned and built monterey (just for the sheer pleasure of watching how everything happens nearly instantly). Then I issued `gulp watch` and started to click on tiles (as if I was not aware about my few hours old issue explaining how there should be a state machine that disables meaningless buttons in any given state).

So, I got this:

![screen shot 2016-07-15 at 5 46 08 pm](https://cloud.githubusercontent.com/assets/2712405/16889753/aaa547a2-4ab6-11e6-8407-888ce8437151.png)

This is of course a problem that needs to be addressed right away, because everyone and his brother will do the same silly thing and claim that monerey is "infested" with bugs
